data/remnant ships.txt

```diff
@@ -348,6 +347,87 @@ ship "Gull" "Gull (Bunks)"
 
 
 
+ship "Ibis"
+	sprite "ship/ibis"
+	thumbnail "thumbnail/ibis"
+	attributes
+		category "Medium Warship"
+		licenses
+			"Remnant Capital"
+		"cost" 30000000
+		"shields" 22000
+		"hull" 11000
+		"required crew" 20
+		"bunks" 48
+		"mass" 300
+		"drag" 7.1
+		"heat dissipation" 0.7
+		"fuel capacity" 700
+		"ramscoop" 2
+		"outfit space" 568
+		"cargo space" 103
+		"weapon capacity" 200
+		"engine capacity" 142
+		"shield generation" 6
+		"shield energy" 4.6
+		"hull repair rate" 2
+		"hull energy" 1.2
+		"outfit scan power" 18
+		"outfit scan speed" 1
+		"tactical scan power" 39
+		weapon
+			"blast radius" 240
+			"shield damage" 2400
+			"hull damage" 1200
+			"hit force" 3000
+	outfits
+		"Inhibitor Cannon" 3
+		"Thrasher Turret" 1
+		"Point Defense Turret" 2
+
+		"Aeon Cell"
+		"Crystal Capacitor" 3
+		"Emergency Ramscoop" 2
+		"Thermoelectric Cooler" 5
+		"Quantum Key Stone"
+		"Salvage Scanner"
+		"Tuning Rifle" 20
+
+		"Forge-Class Thruster"
+		"Crucible-Class Thruster"
+		"Forge-Class Steering"
+		"Crucible-Class Steering"
+		"Scram Drive"
+		
+	drone 0 30 over 
+	drone -34 68 over
+	drone 0 81 over
+	drone 34 68 over
+	fighter -25 10 under
+	fighter 25 10 under
+	engine -16.5 120 0.7
+	engine 0 116.5 1
+	engine 16.5 120 0.7
+	gun 0 -135.5 "Inhibitor Cannon"
+	gun 15.5 -102 "Inhibitor Cannon"
+	gun -15.5 -102 "Inhibitor Cannon"
+	turret 0 -62 "Thrasher Turret"
+	turret 0 -20.5 "Point Defense Turret"
+	turret 0 58.5 "Point Defense Turret"
+	
+	leak "leak" 50 50
+	leak "flame" 50 80
+	leak "big leak" 90 30
+	explode "tiny explosion" 10
+	explode "small explosion" 25
+	explode "medium explosion" 25
+	explode "large explosion" 10
+	"final explode" "final explosion medium"
+	description "This is the Ibis, a multipurpose carrier that serves as a mining platform for terns, as an escort ship for pelican fleets, and a combat platform for terns used in screening roles. It is capable of fielding a single wing of support craft, consisting of two petrels and four terns."
```

Comment:
`This is the <ship>...` isn't the way that any other ship descriptions are worded, instead using `The <ship> is...`
You need to capitalize the names of the other ships (Pelican, Petrel, Tern, Albatross).

Reply:
OK, changed.